src/main.rs

```diff
@@ -1,7 +1,8 @@
+//! These are module-level docs
```

Comment:
UGH, I tried to do this just before the `fn main()` and it was complaining. I found [this SO answer](https://stackoverflow.com/questions/36184407/how-can-i-write-crate-wide-documentation/36184584#36184584) which induced me into thinking the wrong scope of docs. Good to know!

Reply:
The question mark thing also applies to compiler flags (including `#[allow(...)]` statements). If you add an exclamation mark, it applies to the enclosing scope. Without the exclamation mark it applies to whatever is on the next line